modules/commons/src/main/java/org/testcontainers/ext/ScriptUtils.java

```diff
@@ -212,17 +210,16 @@ public static boolean containsSqlScriptDelimiters(String script, String delim) {
 		return false;
 	}
 
-    public static void executeSqlScript(Connection connection, String scriptPath, String script) throws ScriptException {
```

Comment:
By doing that, you break the backward compatibility because the method was public and users might use it

Reply:
I really don't think that somebody uses ScriptUtils from testcontainers library, so it's not even a problem from my perspective.
And btw this class is intended for internal usage:
>Mainly for internal use within the framework.